openpype/hosts/max/plugins/publish/validate_tyflow_data.py

```diff
@@ -73,16 +80,28 @@ def get_tyflow_operator(self, instance):
 
             # There must be at least one animation with export
             # particles enabled
-            has_export_particles = False
             anim_names = rt.GetSubAnimNames(obj)
             for anim_name in anim_names:
                 # get name of the related tyFlow node
                 sub_anim = rt.GetSubAnim(obj, anim_name)
-                # check if there is export particle operator
-                if rt.IsProperty(sub_anim, "Export_Particles"):
-                    has_export_particles = True
-                    break
 
-            if not has_export_particles:
-                invalid.append(member)
+                # Isolate only the events
+                if not rt.isKindOf(sub_anim, rt.tyEvent):
+                    continue
+                has_export_particle = False
+                # Look through all the nodes in the events
+                node_names = rt.GetSubAnimNames(sub_anim)
+                for node_name in node_names:
+                    node_sub_anim = rt.GetSubAnim(sub_anim, node_name)
+                    if rt.hasProperty(node_sub_anim, "exportMode"):
+                        # check if the current export mode of the operator
+                        # is valid for the tycache export.
+                        if node_sub_anim.exportMode != 2 or \
+                            node_sub_anim.exportMode != 3 or \
+                                node_sub_anim.exportMode != 6:
+                            has_export_particle = True
+                            break
+            if not has_export_particle:
+                invalid.append(f"{member} has invalid export Mode.")
```

Comment:
Either:
```suggestion
                invalid.append(f"{member} has invalid Export Mode.")
```
Or:
```suggestion
                invalid.append(f"{member} has invalid export mode.")
```

Otherwise not sure why just Mode is uppercase.